db/create_product.sql

```diff
@@ -1,4 +1,4 @@
 --This query will need to take the 4 parameters defined in the schema and insert a record into the database.
 
-insert into products (Name, Description, Price, Imageurl)
+insert into products (product_name, description, price, url)
```

Comment:
i used lowercase names here for the columns and when i created the table they matched up - i think this is easier.